pages/osx/sips.md

```diff
@@ -0,0 +1,28 @@
+# sips
+
+> Apple Scriptable Image Processing System.
+> Raster/Query images and ColorSync ICC Profiles.
+
+- Specify an output directory so originals does not get modified:
```

Comment:
so _that_ originals _do_ not get modified.

Reply:
Sorry, i'll fix that.